pkg/gql/query/types.go

```diff
@@ -60,10 +60,12 @@ var Header = graphql.NewObject(
 				Type: UnixTimestamp,
 			},
 			"reward": &graphql.Field{
-				Type: graphql.Int,
+				Type:    graphql.Float,
+				Resolve: resolveReward,
 			},
 			"feespaid": &graphql.Field{
```

Comment:
Getting `feespaid` per transactions fails:

```
./transaction.sh 9501 8cad3e5ff99e86ab14702f9987813f7dd7f7052df136696e0d251ceb261ebd07
9501  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   369  100   134  100   235   130k   229k --:--:-- --:--:-- --:--:--  360k
{
    "data": null,
    "errors": [
        {
            "message": "Cannot query field \"feespaid\" on type \"Transaction\".",
            "locations": [
                {
                    "line": 1,
                    "column": 148
                }
            ]
        }
    ]

```

Reply:
FeesPaid in transaction has been removed in favor of GasSpent.

It can be easily calculated at clientSide as GasSpent*GasPrice

Reply:
Awesome!